lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -726,10 +727,18 @@ bool ProxyRenderDelegate::getInstancedSelectionPath(
     // instance, because we don't use instanced draw for single instance render items in order to
     // improve draw performance in Maya 2020 and before.
     const int drawInstID = intersection.instanceID();
+    int instanceIndex = UsdImagingDelegate::ALL_INSTANCES;
 
 #if defined(USD_IMAGING_API_VERSION) && USD_IMAGING_API_VERSION >= 13
     const int usdInstID = drawInstID > 0 ? drawInstID - 1 : 0;
-    SdfPath   usdPath = _sceneDelegate->GetScenePrimPath(rprimId, usdInstID);
+    HdInstancerContext instancerContext;
+    SdfPath usdPath = _sceneDelegate->GetScenePrimPath(rprimId, usdInstID, &instancerContext);
```

Comment:
For point instances, `GetScenePrimPath()` is going to give us back the path to the prototype prim that is being instanced. We want to make sure we capture the instancer context so we can find the `PointInstancer` that is doing the instancing:
https://github.com/PixarAnimationStudios/USD/blob/d8a405a1344480f859f025c4f97085143efacb53/pxr/usdImaging/usdImaging/delegate.h#L389

When multiple levels of instancing are involved, we only ever allow selecting (and eventually manipulating) the top-level instance, at least for now, since making changes at lower levels may have wider-reaching effects.

Reply:
I think you made a change before to allow picking the precise instance from nested instancing. Looks like this is not a desired behavior any more. Could you elaborate on the side effects?